cocos/platform/CCFileUtils.cpp

```diff
@@ -575,7 +575,7 @@ bool FileUtils::writeDataToFile(Data retData, const std::string& fullPath)
     size_t size = 0;
     const char* mode = "wb";
 
-    CCASSERT(!fullPath.empty() && retData.getSize() != 0, "Invalid parameters.");
```

Comment:
maybe you should add another  assert to keep back compatibility.

`CCASSERT(retData.getSize()!=0, "reData can't be size 0")`